src/main/java/org/openRealmOfStars/starMap/Route.java

```diff
@@ -180,10 +113,9 @@ public boolean isFixing() {
    */
   public void makeNextMove() {
     for (int i = 0; i < ftlSpeed; i++) {
-      if (distance > 0) {
-        startX = startX + mx;
-        startY = startY + my;
-        distance--;
+      if (getDistance() > 0) {
+        startX = startX + getMx();
```

Comment:
You made something wrong. You shouldn't mix the old one with the new one.
In your test your route and oldRoute are both Route instance.
I also mada a new branch that compare the old and the new route as you did. The only difference is that I create an OldRoute from your master branch.
https://github.com/sigee/Open-Realms-of-Stars/tree/compare-old-and-new-route
The result is green.